package.json

```diff
@@ -2,58 +2,45 @@
   "name": "final-project",
   "version": "0.1.0",
   "private": true,
-  "prisma": {
-    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts",
-    "resolutions": {
-      "webpack5": "5^"
-    },
-    "scripts": {
-      "dev": "next dev",
-      "build": "next build",
-      "start": "next start",
-      "lint": "next lint",
-      "storybook": "start-storybook -p 6006",
-      "build-storybook": "build-storybook"
-    },
-    "dependencies": {
-      "@next/font": "13.1.6",
-      "@prisma/client": "^4.10.1",
-      "@types/react": "18.0.28",
-      "@types/react-dom": "18.0.11",
-      "cors": "^2.8.5",
-      "dotenv": "^16.0.3",
-      "eslint": "8.34.0",
-      "eslint-config-next": "13.1.6",
-      "fastify": "^4.13.0",
-      "fastify-zod": "^1.2.0",
-      "next": "13.1.6",
-      "postcss-loader": "^7.0.2",
-      "react": "18.2.0",
-      "react-dom": "18.2.0",
-      "zod": "^3.20.6"
-    },
-    "devDependencies": {
-      "@types/node": "^18.13.0",
-      "axios": "^1.3.3",
-      "storybook": "^6.5.16",
-      "@babel/core": "^7.20.12",
-      "@storybook/addon-actions": "^6.5.16",
-      "@storybook/addon-essentials": "^6.5.16",
-      "@storybook/addon-interactions": "^6.5.16",
-      "@storybook/addon-links": "^6.5.16",
-      "@storybook/addon-postcss": "^2.0.0",
-      "@storybook/builder-webpack5": "^6.5.16",
-      "@storybook/manager-webpack5": "^6.5.16",
-      "@storybook/react": "^6.5.16",
-      "@storybook/testing-library": "^0.0.13",
-      "autoprefixer": "^10.4.13",
-      "babel-loader": "^8.3.0",
-      "postcss": "^8.4.21",
-      "prisma": "^4.10.1",
-      "tailwindcss": "^3.2.6",
-      "ts-node": "^10.9.1",
-      "ts-node-dev": "^2.0.0",
-      "typescript": "^4.9.5"
-    }
+  "resolutions": {
```

Comment:
the up-to-date `package.json` is missing the prisma seed script

Reply:
and other dependencies, can you reverse merge from main?

Reply:
I git pull main branch and then get back to this branch to merge main but Storybook doesn't display the tailwind design for the button component and defaults examples/settings appears. Inside of vs code if i reuse the button component in another page, the tailwind class works and  button appears with the tailwind class that i made, how comes?

Reply:
@peelerzin pull the latest storybook changes from the main branch and they should fix this issue.